lib/ansiblelint/__main__.py

```diff
@@ -80,7 +76,12 @@ def main():
         skip.update(str(s).split(','))
     options.skip_list = frozenset(skip)
 
-    playbooks = sorted(set(options.playbook))
+    # no args triggers auto-detection mode
+    if not options.playbook:
```

Comment:
I'd move the comment inside the block to explicitly indicate which branch corresponds to the auto-detection:
```suggestion
    if not options.playbook:
        # no args triggers auto-detection mode
```